verifier/src/report_body.rs

```diff
@@ -339,6 +349,7 @@ impl<E: Accessor<ReportData>> Verifier<E> for ReportDataVerifier {
 mod test {
     use super::*;
```

Comment:
:wrench: To go along with removing extern alloc from lib.rs

```suggestion
mod test {
    extern crate alloc;
    
    use super::*;
```

Reply:
Done.